templates/data/opensearch/index.html

```diff
@@ -0,0 +1,506 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1nVdoKoPrXw8lsvHEtMichlPNfxYU9Uj1zmKowcUXQ_w/edit
+{% endblock meta_copydoc %}
+
+{% block title %}An enterprise  solution for OpenSearch from Canonical{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}data/opensearch{% endblock %}
```

Comment:
What's your for? Isn't the canonical URL this page?

Reply:
Sorry I think I don't fully understand.

Reply:
I think the question is why do you need this block? isn't it already clear what the URL is?

Reply:
Yeah that should be the case, shall I remove it? It was there before I edited the code